### PR TITLE
[ZH] Update `BBCode`

### PR DESCRIPTION
**Pull request by YumeMuzi**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/BBCode/zh.md [hunk 1]
```diff
@@ -1,390 +1,354 @@
...
 # BBCode
 
-![论坛中的编辑器与按钮](img/editor.jpg "论坛中的编辑器")
+**BBCode** 是一种在 osu! 论坛中使用的[标记语言](https://zh.wikipedia.org/wiki/%E7%BD%AE%E6%A0%87%E8%AF%AD%E8%A8%80)。往更大了说，它也用于互联网上的其他绝大多数论坛。它由环绕文本的标记组成，用于表示用于启用富文本的格式、属性、嵌入等。它被用于 osu! 网站的各个地方，比如论坛贴、玩家签名、玩家主页和谱面描述。
 
-**BBCode** 是在 osu!论坛 中使用的一种文本标记语言，也是网络上大多数论坛所支持的文本格式。它是由包围在文本前后的标签所组成的，也可以包含特定的属性。在 osu! 网站上，你可以在帖子、签名和用户页面中使用 BBCode 。
+![论坛中的编辑器与按钮](img/editor.jpg "论坛中的编辑框")
 
-## 注意
+## 表现
```
Comment (RushFTK) [inline wiki/BBCode/zh.md:7]:
```suggestion
## 行为
```
我觉得使用原本的翻译并不会产生歧义。或者说表现让我想起了"performance"，以为后面说的是性能的事。

File: wiki/BBCode/zh.md [hunk 2]
```diff
@@ -1,390 +1,354 @@
----
-outdated_translation: true
----
-
 # BBCode
 
-![论坛中的编辑器与按钮](img/editor.jpg "论坛中的编辑器")
+**BBCode** 是一种在 osu! 论坛中使用的[标记语言](https://zh.wikipedia.org/wiki/%E7%BD%AE%E6%A0%87%E8%AF%AD%E8%A8%80)。往更大了说，它也用于互联网上的其他绝大多数论坛。它由环绕文本的标记组成，用于表示用于启用富文本的格式、属性、嵌入等。它被用于 osu! 网站的各个地方，比如论坛贴、玩家签名、玩家主页和谱面描述。
```
Comment (RushFTK) [inline wiki/BBCode/zh.md:3]:
```suggestion
**BBCode** 是一种在 osu! 论坛中使用的[标记语言](https://zh.wikipedia.org/wiki/%E7%BD%AE%E6%A0%87%E8%AF%AD%E8%A8%80)。往更大了说，它也用于互联网上的其他绝大多数论坛。它由环绕文本的标记组成，用于表示用于启用富文本的格式、属性、嵌入等。在 osu! 网站的各个地方都可以使用BBCode，比如论坛贴、玩家签名、玩家主页和谱面描述。
```

File: wiki/BBCode/zh.md [hunk 3]
```diff
@@ -1,390 +1,354 @@
...
+![论坛中的编辑器与按钮](img/editor.jpg "论坛中的编辑框")
 
-## 注意
+## 表现
 
-虽然帖子编辑器提供了一些基本的格式化工具，但也可以手动编写 BBCode 。BBCode 标签对大小写不敏感。
+不框选文本，直接单击标记按钮时，文本编辑器会在光标前后会插入一组标记。但如果框选了文本，编辑器会在框选文本前后添加标记。
 
-### BBCode 按钮作用
+用户如果希望在文本中使用多种格式的组合，需要将 BBCode 标记嵌套使用。组合时，必须按照标签的**顺序和嵌套**逻辑组合，否则将达不到预期效果。
```
Comment (RushFTK) [inline wiki/BBCode/zh.md:11]:
```suggestion
若要在一整段中组合使用多种格式，用户可以在一段标签内部使用其他标签。然而，使用的顺序和配对必须**一一对应**，错位将导致格式失效。
```

File: wiki/BBCode/zh.md [hunk 4]
```diff
@@ -1,390 +1,354 @@
...
 
-### 组合标签
+- `[centre][b]文本[/b][/centre]` 正确
+- `[b][centre]文本[/b][/centre]` 错误
 
-标签可以组合成更高级或更丰富的版式。但在组合标签时，**必须**遵守标签的嵌套顺序。不遵守嵌套顺序可能会破坏帖子的版式。
+## 标签
 
-例如（注意标签的顺序）：
+像其他许多标记语言一样，BBCode 内格式化文本的标记由一对方括号 (`[]`) 表示。标签分为开始和结束两种，位于需要格式化文本的前后。结束标签中会包含正斜杠 (`/`)。
```
Comment (RushFTK) [inline wiki/BBCode/zh.md:20]:
```suggestion
像其他许多标记语言一样，BBCode 内格式化文本的标记由一对方括号 (`[]`) 表示。标签分为开始和结束两种，区别在于后者包含正斜杠 (`/`)，而前者没有。
```

Comment (RushFTK) [reply wiki/BBCode/zh.md:20]: 在这段没说这段标签会在文本前后。

File: wiki/BBCode/zh.md [hunk 5]
```diff
@@ -1,390 +1,354 @@
...
+`[u]` 下划线标签通过在文本下添加横线，用于强调文本。下划线也会影响粗体或斜体的文本。
 
 ### 删除线
 
-在单词或句子的**中间画一条水平线**。
+```
+[strike]文本[/strike]
+```
+
+`[strike]` 删除线标签用于在文本中间添加横线，标记这段文本将要被移除。
```
Comment (RushFTK) [inline wiki/BBCode/zh.md:62]:
```suggestion
***备注：删除线("strikethrough")英文也称作"strike"***

`[strike]` 删除线标签用于在文本中间添加横线，标记这段文本将要被移除。
```

**Review by RushFTK — APPROVED** [review]
All review are optional and it's ok even without any changes

File: wiki/BBCode/zh.md [hunk 6]
```diff
@@ -1,390 +1,356 @@
...
+[img]图床链接[/img]
+```
+
+`[img]` 图片标签用于在 osu! 论坛中显示在线图片。若要使用这个标签，则用户必须粘贴图片的直链（在 `图床链接` 参数中指定）。本地文件路径（比如 `C:\Users\Name\Pictures\image.jpg`）**将不起作用**。
+
+*注意：网站的 URL **不等同于**图片的直链。*
+
+如果想获得图片的直链，则必须访问图片来源的网站，并将光标悬停在图像上，右键单击，选择“复制图片地址”，之后，将复制好的图片直链粘贴在开始和结束标签之间。
+
+为了获得图像地址，必须导航到其来源的网站，将鼠标悬停在图像上，右键单击图像，然后选择“复制图像地址”。然后，地址应该被复制并粘贴在标签之间。
```
Comment (cl8n) [inline wiki/BBCode/zh.md:248]: I assume one of these was only meant to be a draft or something?

**Review by cl8n — CHANGES_REQUESTED** [review]
don't forget to translate screenshots with text in them -- I added & pushed them here

also try to use better commit names that explain what you are changing.

File: wiki/BBCode/zh.md (hunk 6 above)
Comment (YumeMuzi) [reply wiki/BBCode/zh.md:248]: removing draft paragraph

Comment (YumeMuzi) [reply wiki/BBCode/zh.md]: and ty for the screenshot

**Review by cl8n — APPROVED** [review]

**Pull request merged**